beefy-gadget/Cargo.toml

```diff
@@ -12,6 +12,7 @@ hex = "0.4"
 log = "0.4"
 parking_lot = "0.11"
 thiserror = "1.0"
+wasm-timer = "0.2.5"
```

Comment:
`std::time::Instant` does not work on wasm target, we need this for the browser node.